src/test/javascript/spec/component/example-modeling/example-modeling-submission.component.spec.ts

```diff
@@ -64,7 +64,7 @@ describe('Example Modeling Submission Component', () => {
         correctionStatus: 'CORRECT',
     } as Feedback;
     const mockFeedbackWithoutReference = { text: 'FeedbackWithoutReference', credits: 30, type: FeedbackType.MANUAL_UNREFERENCED } as Feedback;
-    const mockFeedbackInvalid = { text: 'FeedbackInvalid', referenceId: '4', reference: 'reference', correctionStatus: FeedbackCorrectionErrorType.INCORRECT_SCORE };
+    const mockFeedbackInvalid = { text: 'FeedbackInvalid', referenceId: '4', reference: 'reference', correctionStatus: FeedbackCorrectionErrorType.INCORRECT_SCORE } as Feedback;
```

Comment:
not needed, but this activates type checking (in case the feedback model gets changed in the future, you would get a compilation error here, without this this is just "some object" which can have any properties.